### DISCUSSION TITLE
Is it possible to auto update the status based on the value of a checkbox?

### DISCUSSION THREAD

**Discussion by Hitmonsashi** [marcusolsson/obsidian-projects#563] [Help]
Is it possible to auto update the status based on the value of a checkbox? I'd like to automatically update the status front matter value of a note to "Done" if my "Completed" checkbox bool is set to true, is this possible?

**Comment by GH4TP** [comment]
I had the same question. I guess it is a function of creating a new property and setting the property type to checkbox. And in the template using that property with default data, say, 'false' or 'true' depending on your situation.
